### DISCUSSION TITLE
Cannot choose between the following variants of org.jetbrains.kotlinx:kotlinx-html-jvm:0.7.2

### DISCUSSION THREAD

**Discussion by androiddevcoding** [detekt/detekt#3519] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/11345578/110174700-ef5c0700-7e19-11eb-8a74-beeedb4e0e3e.png)
How do I fix this error?

**Comment by cortinico** [comment, marked as answer]
That's hard to answer. 

- Which version of Gradle are you on? (Can you update to the latest?)
- Can you provide a reproducer?

**Reply by androiddevcoding** [reply]
> That's hard to answer.
> 
> * Which version of Gradle are you on? (Can you update to the latest?)
> * Can you provide a reproducer?

Why does this error appear?

 mavenCentral()
    jcenter {
        content {
            // just allow to include kotlinx projects
            // detekt needs 'kotlinx-html' for the html report
            includeGroup "org.jetbrains.kotlinx"
        }
    }
Isn't that a solution to the problem?

Version gradle: 'com.android.tools.build:gradle:4.0.1'
I cannot raise the version.

**Reply by cortinico** [reply]
> Version gradle: 'com.android.tools.build:gradle:4.0.1'
> I cannot raise the version.

That's the version of AGP. I'm asking for the version of Gradle you're running.
You can find it inside `gradle/wrapper/gradle-wrapper.properties`

**Reply by androiddevcoding** [reply]
> > Version gradle: 'com.android.tools.build:gradle:4.0.1'
> > I cannot raise the version.
> 
> That's the version of AGP. I'm asking for the version of Gradle you're running.
> You can find it inside `gradle/wrapper/gradle-wrapper.properties`

Sorry. 
`distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip`

**Reply by cortinico** [reply]
Can you update to Gradle 6.8.3 and retry?
You can update with:
```
./gradlew wrapper --gradle-version 6.8.3 --distribution-type bin
```

**Reply by androiddevcoding** [reply]
> Can you update to Gradle 6.8.3 and retry?
> You can update with:
> 
> ```
> ./gradlew wrapper --gradle-version 6.8.3 --distribution-type bin
> ```

![image](https://user-images.githubusercontent.com/11345578/110467614-ab565400-80f0-11eb-9afa-e6b8f0326528.png)
No worked.

**Reply by cortinico** [reply]
Then I need to ask you for a reproducer project. I'm unable to help you further if you can't share your whole project.

**Reply by androiddevcoding** [reply]
I had a version:
`    ext.kotlin_version = '1.3.71'`
Updated to:
`    ext.kotlin_version = '1.4.10'`
Everything worked, sorry to bother you
